app/src/main/res/values/settings_keys.xml

```diff
@@ -523,6 +524,7 @@
         <item>Nynorsk</item>
         <item>O‘zbek</item>
         <item>Polski</item>
+        <item>ଓଡ଼ିଆ</item>
```

Comment:
You have swapped Polish and Odia. Please move this line up.

Reply:
Okay

Reply:
> You have swapped Polish and Odia. Please move this line up.

I have performed the changes suggested by you.